src/main/target/link/stm32_flash_f303_128k.ld

```diff
@@ -14,7 +14,8 @@ MEMORY
 {
   FLASH  (rx)     : ORIGIN = 0x08000000, LENGTH = 126K /* last 2kb used for config storage */
   RAM    (xrw)    : ORIGIN = 0x20000000, LENGTH = 40K
+  CCM    (xrw)    : ORIGIN = 0x10000000, LENGTH = 8K
   MEMORY_B1 (rx)  : ORIGIN = 0x60000000, LENGTH = 0K
 }
 
-INCLUDE "stm32_flash.ld"
+INCLUDE "stm32_flash_ccm.ld"
```

Comment:
It's possible to use C preprocessor on linker file first. All goodies from C preprocessor are available then (memory sizes from Makefile, target, ...).

https://github.com/ledvinap/cleanflight/blob/local/Makefile#L890
It is possbile without grep, but `-ggdb3` must be removed from `$(CFLAGS)` for this gcc invocation

There was discussion about this, version with minimal target-specific files was used. Personally I prefer preprocessor (everything is in one place), but that is matter of taste